CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.2.0] - 2024-10-16
+### Changed
+- Updated all dependency versions for hive, hadoop, datanucleus, reflections, guava, tez, mockito, junit and added them in pom.xml <properties> section
+- Updated all maven plugin versions
+- Adjusted code for Java21 and removed Guava Optional from the code
```

Comment:
Maybe be more specific with what was changed for Java 11 and put the Guava Optional change on its own line.

Reply:
done